packages/services/rate-limit/package.json

```diff
@@ -11,13 +11,15 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@graphql-yoga/logger": "0.0.1-alpha-20230303125454-5d621255",
```

Comment:
let's avoid using alpha if it's already released

Reply:
It hasn't been released yet. I'll update it after we release.

Reply:
Done!